packages/components/src/common/styles-dictionary/scss/_variables.scss

```diff
@@ -379,206 +379,179 @@ $sds-color-semantic-positive-surface-secondary-dark: #85d898;
 $sds-color-semantic-positive-text: #105b2b;
 $sds-color-semantic-positive-text-dark: #bcecc5;
 $sds-corner-l: 20px;
-$sds-corner-l-dark: 20px;
 $sds-corner-m: 4px;
-$sds-corner-m-dark: 4px;
 $sds-corner-s: 2px;
-$sds-corner-s-dark: 2px;
 $sds-corner-none: 0px;
-$sds-corner-none-dark: 0px;
 $sds-drop-shadow-l: 0.0px 2.0px 12.0px 0px rgba(0, 0, 0, 0.3);
-$sds-drop-shadow-l-dark: 0.0px 2.0px 12.0px 0px rgba(0, 0, 0, 0.3);
 $sds-drop-shadow-m: 0.0px 2.0px 10.0px 0px rgba(0, 0, 0, 0.15), 0.0px 2.0px 4.0px 0px rgba(0, 0, 0, 0.15);
-$sds-drop-shadow-m-dark: 0.0px 2.0px 10.0px 0px rgba(0, 0, 0, 0.15), 0.0px 2.0px 4.0px 0px rgba(0, 0, 0, 0.15);
 $sds-drop-shadow-s: 0.0px 2.0px 4.0px 0px rgba(0, 0, 0, 0.25);
-$sds-drop-shadow-s-dark: 0.0px 2.0px 4.0px 0px rgba(0, 0, 0, 0.25);
 $sds-drop-shadow-none: none;
-$sds-drop-shadow-none-dark: none;
 $sds-font-color: black;
-$sds-font-color-dark: black;
+$sds-font-color-narrow: black;
```

Comment:
I probably just don't have the context lol But this replaces `dark` with `narrow` - is it gonna break dark mode or?

Reply:
Oh no, the `-dark` style was unnecessary and needed to be removed! we don’t need dark mode specific values for fonts since dark mode is only for `colors` and `borders`. The new `narrow` styles have been added to the fonts, though. Hope that makes sense!

As for this specific design token, `font-color`, I should probably remove it, since we should only use the text colors defined in the semantic colors, not font-color. Thanks for pointing that out! 😄

Reply:
Ahh that makes sense lol Thanks so much for the clarification 💡 LGTM!!